### PR TITLE
Add closure style

### PR DESCRIPTION
**Pull request by initFabian**
Add a style guideline for closure to promote consistency.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -413,6 +415,32 @@ attendeeList.sort { a, b in
 }
 ```
 
+To condense the closure even further, remove any `parentheses` and unless the closure returns something other than `Void`, remove `arrows(->)` as well.
```
Comment (sallyransom) [inline README.markdown]: use the following to encase the return arrow operator:

"...remove the return arrow (`->`)..."

File: README.markdown [hunk 2]
```diff
@@ -413,6 +415,32 @@ attendeeList.sort { a, b in
...
+
+**Preferred:**
+
+```swift
+let action = UIAlertAction(title: "Cancel", style: .Cancel) { alertAction in
```
Comment (sallyransom) [inline README.markdown]: show a preferred example of a closure with a return value as well

File: README.markdown [hunk 3]
```diff
@@ -413,6 +415,32 @@ attendeeList.sort { a, b in
...
+    //Code
+}
+```
+
+**Not Preferred:**
+
+```swift
+let action = UIAlertAction(title: "Cancel", style: .Cancel) { (alertAction) -> () in
```
Comment (sallyransom) [inline README.markdown]: show a not preferred example of a closure with a return value as well

**Comment by sallyransom** [comment]
@fbuentello code review done. i give it a thumbs up. it'd be nice to get code reviews and votes from @JosephSouthern @yfujiki @tLewisII  and finally @iovortex as well

**Comment by JosephSouthern** [comment]
Looks good to me.

File: README.markdown (hunk 1 above)
Comment (initFabian) [inline README.markdown]: Done

File: README.markdown (hunk 2 above)
Comment (initFabian) [inline README.markdown]: Done

File: README.markdown (hunk 3 above)
Comment (initFabian) [inline README.markdown]: Done

**Comment by initFabian** [comment]
@sallyransom Addressed PR issues, If you can think of better examples please don't hesitate to tell me.

**Comment by yfujiki** [comment]
LGTM 👍

**Comment by tLewisII** [comment]
Looks good to me as well. 👍

**Comment by iovortex** [comment]
Awesome. Congrats @fbuentello. Merging.

**Pull request merged**